defaults.env

```diff
@@ -3,3 +3,4 @@ PULSE_DOMAIN=www.mozillapulse.org
 NETWORK_API_DOMAIN=network.mofoprod.net
 VIRTUAL_ROOT=/
 TARGET_DOMAIN=network.mofostaging.net
+CSP=default-src 'self'; script-src 'self' cdn.optimizely.com; style-src 'self' code.cdn.mozilla.net fonts.googleapis.com; img-src * data:; font-src 'self' fonts.gstatic.com fonts.googleapis.com code.cdn.mozilla.net; connect-src 'self' *.mozilla.org; media-src 'self'; object-src 'self'; child-src 'self'; frame-src 'self';
```

Comment:
Haha you're right about frame-src - it was deprecated in v2, undeprecated in v3 draft - but Firefox still complains that it's deprecated!